src/index.html

```diff
@@ -7,6 +7,359 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="wrapper">
+      <header class="header wrapper__header">
+        <div class="header__container">
+          <a href="#" class="header__logo">
+            <img src="./images/logo.png" alt="logo">
+          </a>
+          <nav class="header__menu menu-header">
+            <ul class="menu-header__list">
+              <li class="menu-header__item">
+                <a href="#apple" class="menu-header__link menu-header__link--is-active is-active">Apple</a>
+              </li>
+
+              <li class="menu-header__item">
+                <a href="#samsung" class="menu-header__link">Samsung</a>
+              </li>
+
+              <li class="menu-header__item">
+                <a href="#smartphones" class="menu-header__link">Smartphones</a>
+              </li>
+
+              <li class="menu-header__item">
+                <a
+                  href="#laptops"
+                  class="menu-header__link"
+                  data-qa="nav-hover"
+                >
+                  Laptops & Computers
+                </a>
+              </li>
+
+              <li class="menu-header__item">
+                <a href="#gadgets" class="menu-header__link">Gadgets</a>
+              </li>
+
+              <li class="menu-header__item">
+                <a href="#tablets" class="menu-header__link">Tablets</a>
+              </li>
+
+              <li class="menu-header__item">
+                <a href="#photo" class="menu-header__link">Photo</a>
+              </li>
+
+              <li class="menu-header__item">
+                <a href="#video" class="menu-header__link">Video</a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </header>
+      <main class="main wrapper__main">
```

Comment:
```suggestion
      </header>
      
      <main class="main wrapper__main">
```

Reply:
@valet2102  Why did you decide not to update this?